flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -383,7 +383,9 @@
 
 	/**
 	 * Whether to use the LargeRecordHandler when spilling.
+	 * @deprecated use {@link AlgorithmOptions#USE_LARGE_RECORDS_HANDLER}
```

Comment:
Can't we remove them right away? This is such an obscure feature. But it's also fine like this.

Reply:
I'd prefer to leave it for now. Otherwise we should remove remaining options which might require quite some work in the DataSet codebase.

Reply:
Gotcha 👌